index.js

```diff
@@ -0,0 +1,184 @@
+            baseUrl: this.baseUrl,
+            headers: {
+                [data.crumbRequestField]: data.crumb
+            }
+        });
+
+        if (jenkinsClient) return callback(null, jenkinsClient);
+
+        return callback(new Error('Failed to instantiate jenkins client'));
+    }
+
+    /**
+     * Get build number of the job's last build
+     * @method getBuildNumber
+     * @param  jobName                      name of the jenkins job and also buildID in config
+     * @param  {Function} callback          Callback with error, jenkins client object and build number
+     */
+    getBuildNumber(jobName, jenkinsClient, callback) {
+        const get = jenkinsClient.job.get.bind(jenkinsClient.job);
+
+        return get(jobName, (err, data) => {
+            try {
+                callback(err, jenkinsClient, data.lastBuild.number);
+            } catch (e) {
+                callback(e);
+            }
+        });
+    }
+
+    /**
+     * Create a jenkins job and start the build
+     * @method _start
+     * @param  {Object}   config            A configuration object
+     * @param  {String}   config.buildId    ID for the build and also name of the job in jenkins
+     * @param  {String}   config.jobId      ID for the job
+     * @param  {String}   config.pipelineId ID for the pipeline
+     * @param  {String}   config.container  Container for the build to run in
+     * @param  {String}   config.scmUrl     Scm URL to use in the build
+     * @param  {Function} callback          Callback with null if successful otherwise error
+     */
+    _start(config, callback) {
+        const fakeConfigPath = path.resolve(__dirname, './config/test-job.xml');
+        const xml = fs.readFileSync(fakeConfigPath, 'utf-8');
+
+        async.waterfall([
+            this.getCrumb.bind(this),
+            this.initJenkinsClient.bind(this),
+            (jenkinsClient, cb) => {
+                const create = jenkinsClient.job.create.bind(jenkinsClient.job);
+
+                return create(config.buildId, xml, (err) => {
+                    cb(err, jenkinsClient);
+                });
+            },
+            (jenkinsClient, cb) => {
+                const build = jenkinsClient.job.build.bind(jenkinsClient.job);
+
+                return build(config.buildId, cb);
+            }
+        ], (error) => {
+            if (error) return callback(new Error(error.message));
+
+            return callback(null);
+        });
+    }
+
+    /**
+     * Stop the build
+     * @method stop
+     * @param  {Object}   config            A configuration object
+     * @param  {String}   config.buildId    ID for the build and also name of the job in jenkins
+     * @param  {Function} callback          Callback with null if successful otherwise error
+     */
+    _stop(config, callback) {
+        async.waterfall([
+            this.getCrumb.bind(this),
+            this.initJenkinsClient.bind(this),
+            /* eslint-disable arrow-body-style */
```

Comment:
Alternatively, you can write it like:

```
(jenkinsClient, cb) => this.getBuildNumber(config.buildId, jenkinsClient, cb);
```

That should prevent the ESLint check from complaining